vb-be/src/modules/room/room.controller.ts

```diff
@@ -21,39 +22,21 @@ export class RoomController {
 
     try {
       if (!roomObj) {
-        throw new ErrorHandler(HttpStatus.Error, ErrorText.InvalidDataSet);
+        throw new ErrorHandler(HttpStatus.Error, ErrorText.Forbidden);
```

Comment:
I think this should be invalid dataset, since no room object means the correct parameters weren't supplied

Reply:
yeah lol my bad i forgot to change it